.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,26 @@
+// Dev Container for Rust, website, prql-js and prql-python
+{
+  "image": "ghcr.io/prql/prql-devcontainer-base:latest",
+  "features": {
+    "ghcr.io/devcontainers/features/hugo:1": {},
+    "ghcr.io/devcontainers/features/python:1": {},
+    "ghcr.io/devcontainers/features/node:1": {}
+  },
+  "customizations": {
+    "vscode": {
+      "extensions": [
+        "budparr.language-hugo-vscode",
+        "mitsuhiko.insta",
+        "prql-lang.prql-vscode",
+        "esbenp.prettier-vscode"
```

Comment:
If this supports comments, let's link to https://github.com/PRQL/prql/blob/e132c3de9b1099f0fa0d7929c921a301eaccccc4/.vscode/extensions.json (and link back from there to here) so we keep them all in sync. We should add `rust-analyzer` here too

Reply:
I think I commented on this before, but did not write it here because the `rust-analizer` is already included in the base image metadata.

However, adding it here is not a problem, so let's add it.